lib/internal/Magento/Framework/Logger/Test/Unit/MonologTest.php

```diff
@@ -20,23 +22,9 @@ public function testAddRecord()
 
         $logger->pushHandler($handler);
 
-        $logger->addError('test');
+        $logger->addRecord(Logger::ERROR, 'test');
         list($record) = $handler->getRecords();
 
         $this->assertSame('test', $record['message']);
     }
-
-    public function testAddRecordAsException()
```

Comment:
Why was it removed ?

Reply:
Add record accepts only string in the new monolog version